### PR TITLE
Added submit logic

### PR DESCRIPTION
**Pull request by dstrizzi**
Added submit logic to keep a player from adding more than one shape per expedition card

### CODE REVIEW

**Comment by dstrizzi** [comment]
I don't know if it's a new bug but I'm noticing that if you submit a shape on the 7th of 7 expedition cards, that phase counter will not increase.

**Comment by bmorelli25** [comment]
> I don't know if it's a new bug but I'm noticing that if you submit a shape on the 7th of 7 expedition cards, that phase counter will not increase.

That is a new bug. I'll try and figure it out.

**Comment by dstrizzi** [comment]
This has been fixed in a convoluted way.

**Comment by bmorelli25** [comment]
Removed conflicts in my recent commit. Now looking into functionality.

File: src/shapes/Shapes.js [hunk 1]
```diff
@@ -12,10 +12,16 @@ export const validateShape = (selection, expedition) => {
...
+    });
   }
-  
+  // for (i = 0; i < cleanedSelection.length; i++){
+  //   cleanedSelction[i].reduce((acc, curr) => acc+curr)
+  // }
+
+  let skipShapeTurn = cleanedSelection.flatMap((x) => x);
+  skipShapeTurn = skipShapeTurn.reduce((acc, curr) => acc + curr);
+  if (skipShapeTurn === 1) return true;
```
Comment (bmorelli25) [inline src/shapes/Shapes.js:22]: What is this for?

File: src/Players.js [hunk 2]
```diff
@@ -2,18 +2,36 @@ import React, { useContext, useState } from "react";
...
+  const handlePlayerSubmits = () => {
+    roundSubmits++;
+    playerSubmit++;
+    state.roundSubmits = roundSubmits;
+    if (roundSubmits == state.numPlayers) {
+      console.log("all players have submitted");
+      roundSubmits = 0;
+    }
+  };
+
```
Comment (bmorelli25) [inline src/Players.js]: It seems weird to me that this function handles both player and round submits

File: src/Players.js [hunk 3]
```diff
@@ -2,18 +2,36 @@ import React, { useContext, useState } from "react";
...
     if (p === 3) hand = state.playerFour; // player four
     return hand;
-  }
+  };
+
+  // checks and adds number of submits per expedition card round
+  const handlePlayerSubmits = () => {
+    roundSubmits++;
+    playerSubmit++;
+    state.roundSubmits = roundSubmits;
```
Comment (bmorelli25) [inline src/Players.js]: We can't run with this method of updating state, unfortunately. This is just waiting to break in our faces

**Comment by dstrizzi** [comment]
Should've commented it out. It allows a user to submit one square when they
cannot (or don't want to) use the shape provided

File: src/Players.js (hunk 3 above)
Comment (bmorelli25) [reply src/Players.js]: I see what you mean about state not updating though

File: src/shapes/Shapes.js (hunk 1 above)
Comment (bmorelli25) [reply src/shapes/Shapes.js:22]: Ahh, makes sense

**Pull request merged**